driver-core/src/main/com/mongodb/internal/operation/ListCollectionsOperation.java

```diff
@@ -206,6 +226,7 @@ private BsonDocument getCommand() {
         if (nameOnly) {
             command.append("nameOnly", BsonBoolean.TRUE);
         }
+        putIfTrue(command, "authorizedCollections", authorizedCollections);
```

Comment:
As discussed, add a unit test for this.

Reply:
Done.

The added `ListCollectionsOperationTest` tests the public method `ListCollectionsOperation.execute` instead of testing the (currently private) method `getCommand`. This way it covers more of `ListCollectionsOperation`. Since mocking is needed for this, I introduced `MongoMockito.mock` (see its documentation for the detailed explanation and links to demo tests). I demonstrated it to @vbabanin and @rozza, and my understanding was that they liked this improvement over `Mockito.mock`.